docs/architecture/diagrams/weather.gov software stack.puml

```diff
@@ -0,0 +1,45 @@
+@startuml weather.gov software stack
```

Comment:
yay this is a great start! can update with our other user types when we setup authentication.